docs/jailer.md

```diff
@@ -188,15 +185,16 @@ Finally, the jailer switches the `uid` to `123`, and `gid` to `100`, and execs
 ```
 ./firecracker \
   --id="551e7604-e35c-42b3-b825-416853441234" \
-  --api-sock=/api.socket \
   --start-time-us=<opaque> \
   --start-time-cpu-us=<opaque>
 ```
 
-We can now use the socket at
-`/srv/jailer/firecracker/551e7604-e35c-42b3-b825-416853441234/root/api.socket`
+Now firecracker creates the socket at
+`/srv/jailer/firecracker/551e7604-e35c-42b3-b825-416853441234/root/<api-sock>`
 to interact with the VM.
 
+Note: default value for <api-sock> is `/tmp/firecracker.socket`
```

Comment:
This needs to be updated as well to `/run/firecracker.socket`.

Reply:
changed 👍